src/models/spaces/charts/chart-data-set.ts

```diff
@@ -166,13 +194,19 @@ export const ChartDataSetModel = types
       self.dataPoints.splice(0, self.dataPoints.length);
     }
 
+    function setMaxDataPoints(maxPoints: number) {
+      self.maxPoints = maxPoints;
+    }
```

Comment:
With the maxPoints function above (and also eventually if you allowed setting fixedMin and fixedMax) there's no way to unset the value once set, right?

The behavior of the graph with this is unset (showing all data) may be a desirable one to switch to, but I don't think you can switch back to it once you switch away.

Reply:
Oh, actually I see that you could switch to auto-scaling the x-axis if maxPoints were zero. This is slightly hidden.

Maybe one option would be to have the type of maxPoints be `types.optional(types.number, 0)`, to show there is a default. Not sure if that is any more self-documenting.